### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by vixentael**
Updated readme according to 0.90.0 release.

Major changes:

1. Update features table, add API form-factors, update cryptography & dev-friendly tables.
2. Slightly cleaned up licensing, regulations.
3. Remove different example projects in a favour of Acra Engineering Examples.
4. Update Installation – link to the doc server.
5. Update Availability – mention the most important things, link to the doc server for rest.
6. Completely rewrite "how acra works", befriend with doc server.
7. Update architecture images, update animation (should we drop it?)

In a nutshell, removed things that are not important now, add things that appeared, and link to the docs for everything.

Bonus point: removed everything from wiki, only 'home' and images left.

It's easier to review the Readme IRL than reading the markdown https://github.com/cossacklabs/acra/blob/vxxtl/readme/README.md

### CODE REVIEW

**Review by shadinua — APPROVED** [review]
Great!

**Review by Lagovas — APPROVED** [review]

**Review by Lagovas — APPROVED** [review]
thanks, it has become really more simple to read
